src/utils/quote_report_utils.ts

```diff
@@ -67,6 +65,7 @@ export function publishQuoteReport(
 ): void {
     if (kafkaProducer && KAFKA_TOPIC_QUOTE_REPORT) {
         const extendedQuoteReport: ExtendedQuoteReport = {
+            ammQuoteUniqueId: logOpts.submissionBy === 'gaslessSwapAmm' ? logOpts.ammQuoteUniqueId : undefined,
```

Comment:
Question: do we need to change the Kafka schema definition if we're adding a new field?